### ISSUE TITLE
[BUG] Egeria Intellij build failure

### ISSUE THREAD

**Issue by MihaiIliescu** [odpi/egeria#7324]
### Current Behavior

I pulled the up to date egeria repo from main and after the maven build I tried to run the build from the intellij interface and I got an error: 

~/egeria/open-metadata-test/open-metadata-fvt/open-types-fvt/open-types-test/src/test/java/org/odpi/openmetadata/fvt/opentypes/server/TestBeansToAccessOMRS.java:126766:36
java: variable primitivePropertyValueForMap is already defined in method testCreateAgreementItem()

The class that fails is generated code and declares multiple times the same variable.

The issue is not only on my setup, I checked with a colleague and we got the error.

### Expected Behavior

The build finishes successfully.

### Environment

```markdown
- Egeria: 3.15-SNAPSHOT
- OS: MacOS Monterey v12.6.2 
- Java: 11
- Browser (for UI issues): -
- Additional connectors and integration: -
- Intellij Version my machine: IntelliJ IDEA 2022.3.1 (Ultimate Edition)
- Intellij Version colleague machine: IntelliJ IDEA 2021.1.3 (Ultimate Edition)
```

**Comment by davidradl** [comment]
Locally I can build core Egeria and get a valid generated file  TestBeansToAccessOMRS.java.
I can confirm that the IJ build creates an invalid version of TestBeansToAccessOMRS.java with duplicate declarations.

**Comment by planetf1** [comment]
I
 - cloned the egeria repo (with up to date main) - so completely clean
 - did a 'mvn clean install' at CLI
 - Opened the directory with intellij ie as a new project
 - Answered 'maven' when prompted for maven+gradle
 - Let IntelliJ index etc
 - Clicked 'build project'
 - Clicked ' enable annotation processing' when prompted
 
 The resultant build was clean
 
 I inspected the file, and at line 126766 of TestBeansToAccessOMRS.java I see
 ```
 MapPropertyValue testMapPropertyValue = null;
 ```
 which no errors - but also different content to the error you are reporting.
 
 I had left my JVM at default. On checking, I am using 'temurin 17' - and the language level defaulted to 17 (this is something I usually check/change after importing a project
 I also checked in settings under Maven, and I am running the Bundled maven 3 (3.8.1)
 
 @davidradl @MihaiIliescu is the sequence I followed exactly the same as you?

**Comment by planetf1** [comment]
Under Help->Custom properties, I also set 'idea.max.intellisense.filesize=10000' which allowed IntelliJ to analyze even these large source files (mem/cpu usage higher) which clarifies these issues in the file
<img width="1216" alt="Screenshot 2023-01-26 at 12 52 29" src="https://user-images.githubusercontent.com/7292002/214840214-aba4d0b2-31dc-45ae-945a-8d9452a04aae.png">

**Comment by planetf1** [comment]
For the gradle build, no code has been added to build/execute these tests at all - I missed this when doing the port
In the case of maven whilst test code is built, there is no execution. Looking at maven.

**Comment by davidradl** [comment]
I think I have found a bug in the generator - just testing.

**Comment by planetf1** [comment]
I'm looking at why the tests don't run in our build. I can't see anything changed in maven for years .
However with our upcoming move to gradle I'll focus my fix there. Just experimenting at the moment - WITHOUT your fix, as first I went to see a compile failure! Then I will try together with your fix.

**Comment by davidradl** [comment]
I will not merge until you give me the nod. @MihaiIliescu you could try applying the pr locally and test it works if you like

**Comment by planetf1** [comment]
I have fixed up gradle to actually build the tests.
Next I will incorporate the code fix, and verify the tests run.

**Comment by MihaiIliescu** [comment]
> 

yes, I'll test it now

**Comment by MihaiIliescu** [comment]
Just finished testing #7325.

Pulled David's PR,  ran mvn clean install, and then build from Intellij with no errors. Great work!

**Comment by planetf1** [comment]
There's a new PR https://github.com/odpi/egeria/pull/7327 which incorporates David's fix, but also corrects the gradle build to actually run the tests!

It was ready to merge, but blocked by many code style warnings from codeQL. Awaiting a decision as to what to do with those before we merge.

with the pending change to v4 I suggest this waits until after that (Monday 6 Feb)

**Comment by planetf1** [comment]
@mandy-chessell @dwolfson

**Comment by planetf1** [comment]
Though not directly related, I am intrigued if https://github.com/odpi/egeria/issues/7425 would make any difference

**Comment by planetf1** [comment]
This should be fixed now, though note issue #7536
